lib/go_script/go.rb

```diff
@@ -40,8 +40,9 @@ def version
   end
 
   def exec_cmd(cmd)
-    status = system(
-      { 'RUBYOPT' => nil }, cmd, err: :out)
+    env = {}.merge(ENV)
+    env.delete('RUBYOPT')
+    status = system(env, cmd, err: :out)
```

Comment:
Better yet... We don't need it anymore. Just realized that since none of the commands start with `bundle exec` anymore (or shouldn't...I _could_ make sure to strip any `bundle exec` prefixes from `cmd` here), given the `./go` script preamble generated from `GoScript::Template`, this filtering is no longer necessary. Deleted.